### PR TITLE
:zap: NEW: ensure working also on non public schema

### PR DESCRIPTION
**Pull request by jycouet**

### CODE REVIEW

File: projects/core/postgres/schema-builder.ts [hunk 1]
```diff
@@ -104,6 +104,17 @@ export class PostgresSchemaBuilder {
...
 
+  private schemaOnly(e: EntityDbNamesBase) {
+    if (e.$entityName.includes('.')) {
+      return e.$entityName.split('.')[0]
+    }
+    if (this.specifiedSchema) {
+      return this.specifiedSchema
+    }
+    // Should default to `public`
+    return 'public'
```
Comment (noam-honig) [inline projects/core/postgres/schema-builder.ts:115]: I think it's better to  use the `specified schema` parameter that is received in the constructor

Comment (noam-honig) [reply projects/core/postgres/schema-builder.ts:115]: While you are on it - I think it's also worth while to fix the `whereTableAndSchema` to always include the schema in the where

Comment (jycouet) [reply projects/core/postgres/schema-builder.ts:115]: You mean to replace all this by: 

```ts
constructor(
    private pool: SqlDatabase,
    schema?: string,
  ) {
      this.specifiedSchema = schema ?? 'public'
  }
```

?

Comment (noam-honig) [reply projects/core/postgres/schema-builder.ts:115]: Sounds about right - it requires a bit of testing - but yes

**Pull request merged**